### PR TITLE
Reintroduce the `Email` pack category

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5348)

## Description
_Part one:_
 Reintroduce the `Email` pack category. packs to include are Email gateways, email packs. e.g: Trustwave, CISCO ESA, Cisco SMA, Proofpoint TAP, Symantec Messaging, Gateway, MimeCast, FireEye EX, EWS O365, EWS v2, Gmail, POP3, Gmail Single User, MS Graph Mail, ArmorBlox, EmailCommunication, Exchange 2016 Complaince, IronScales, MailSender(New), PerceptionPoint.

## Minimum version of Cortex XSOAR
- [X] 6.0.0

## Does it break backward compatibility?
   - [X] No

### CODE REVIEW

File: Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-category: Data Enrichment & Threat Intelligence
+category: Email
```
Comment (cforbes-tw) [inline Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.yml:1]: The Trustwave Fusion Integration is not an email integration.
TrustwaveSEG is Trustwave's Secure Email Gateway product.

Comment (maimorag) [reply Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.yml]: Trustwave Fusion category will remain Data Enrichment & Threat Intelligence, it was removed from this pr.

**Comment by maimorag** [comment]
After consulting with @ostolero EwsExtension test playbook will remain skipped therefore there is a validation error: 
Packs/MicrosoftExchangeOnline/Integrations/EwsExtension/EwsExtension.yml - [CJ105]

File: Packs/MicrosoftGraphFiles/ReleaseNotes/1_1_10.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### O365 File Management (Onedrive/Sharepoint/Teams)
+Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/MicrosoftGraphFiles/ReleaseNotes/1_1_10.md:5]: Please add here the docker image update as well

File: Packs/Mimecast/ReleaseNotes/2_1_6.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Mimecast v2
+Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/Mimecast/ReleaseNotes/2_1_6.md:5]: Add here also the docker image update

File: Packs/Symantec_Messaging_Gateway/ReleaseNotes/1_0_12.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Symantec Messaging Gateway
+Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/Symantec_Messaging_Gateway/ReleaseNotes/1_0_12.md:5]: Same as a above

File: Tests/Marketplace/approved_categories.json [hunk 5]
```diff
@@ -15,6 +15,7 @@
 	"Messaging",
 	"Network Security",
 	"Utilities",
-	"Vulnerability Management"
+	"Vulnerability Management",
+	"Email"
```
Comment (Shellyber) [inline Tests/Marketplace/approved_categories.json:19]: You've already added that to the approved categories no?
Please merge from master and verify

**Review by Shellyber — CHANGES_REQUESTED** [review]
Nice.
Please pay more attention to the docker update release notes

**Comment by maimorag** [comment]
regading valdiation error of Packs/MicrosoftExchangeOnline/Integrations/EwsExtension/EwsExtension.yml - [DO106],
from the yml :
dockerimage: demisto/powershell-ubuntu:7.1.4.24032
 # Please do not change the docker image until CIAC-5623 is handled.

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_0_9.md [hunk 6]
```diff
@@ -3,4 +3,4 @@
 ##### O365 - EWS - Extension
 - Downgraded the PowerShell version used by the integration to prevent the RPS deprecation banner from breaking the 
   outputs of command executions.
-- Fixed an issue where the integration description indicated the wrong command to test successful authentication.
\ No newline at end of file
+- Fixed an issue where the integration description indicated the wrong command to test successful authentication.
```
Comment (Shellyber) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_0_9.md:6]: Why this change?
revert this file

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_0_10.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+
+#### Integrations
+
+##### O365 - EWS - Extension
+- Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_0_10.md:5]:
```suggestion
Updated the pack category to *Email*.
```

File: Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py [hunk 8]
```diff
@@ -62,16 +62,16 @@ def prepare_headers(self, uri: str):  # pragma: no cover
             date=request_date,
             req_id=request_id,
             uri=uri,
-            app_key=self.options.app_key
+            app_key=self.options.app_key  # type: ignore[attr-defined]
```
Comment (Shellyber) [inline Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py:65]: Why the changes here?
Please revert

Comment (Shellyber) [reply Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py:65]: What about this? @maimorag

**Review by Shellyber — COMMENTED** [review]
Fix the docker image validation also please

File: Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py (hunk 8 above)
Comment (maimorag) [reply Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py:65]: it causes an error without it:
in the python file, there is an object(mimecastOptions) that is inherence from IntegrationOptions, but mypy doesn't recognize the fields that are in Mimecast options that were inherent with pydentic

Comment (maimorag) [reply Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py:65]: probably new validation were added since it was merged

**Review by Shellyber — APPROVED** [review]
